superduperdb/backends/local/compute.py

```diff
@@ -0,0 +1,59 @@
+import typing as t
+
+from superduperdb import logging
+from superduperdb.backends.base.compute import ComputeBackend
+
+
+class LocalComputeBackend(ComputeBackend):
+    """
+    A mockup backend for running jobs locally.
+    """
+
+    def __init__(
+        self,
+    ):
+        self.futures_collection: t.Dict[str, t.Any] = {}
+
+    def name(self) -> str:
+        return "Local"
+
+    def submit(self, function: t.Callable, **kwargs) -> None:
+        """
+        Submits a function for local execution.
+
+        :param function: The function to be executed.
+        :param kwargs: Additional keyword arguments to be passed to the function.
+        """
+        future = function(**kwargs)
+        self.futures_collection[future.key] = future
```

Comment:
This will fail.. 
future.key?
local functions are synchronous calls without any futures

In python, developers will be confused here 
thinking this is Async call

because this is valid in async subroutines.